src/sass/shared/mixins.scss

```diff
@@ -132,3 +144,12 @@
         box-shadow: $shadow-color-pill-button-white-hover;
     }
 }
+
+@mixin valign-flex-center($horizontalAlign: false) {
```

Comment:
because it ads an clear abstraction level, useful since vertical centering is always a pain
But I'll remove it if you prefer, of course.